src/group_inst.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CYCAMORE_SRC_MANAGER_INST_H_
+#define CYCAMORE_SRC_MANAGER_INST_H_
+
+#include "cyclus.h"
+#include "cycamore_version.h"
+
+namespace cycamore {
+
+/// @class GroupInst
+/// @section introduction Introduction
+/// @section detailedBehavior Detailed Behavior
+/// @warning The GroupInst is experimental
+class GroupInst
+    : public cyclus::Institution,
+      public cyclus::toolkit::CommodityProducerManager,
+      public cyclus::toolkit::Builder {
+ public:
+  /// Default constructor
+  GroupInst(cyclus::Context* ctx);
+
+  /// Default destructor
+  virtual ~GroupInst();
+
+  virtual std::string version() { return CYCAMORE_VERSION; }
+
+  #pragma cyclus
+
+  #pragma cyclus note {"doc": "An institution that owns and operates a " \
+                              "manually entered list of facilities in " \
+                              "the input file"}
```

Comment:
This is the same description that is in manager_inst.  Some clarification of about what distinguishes the two would be helpful.